packages/client/src/App.tsx

```diff
@@ -97,358 +97,381 @@ function createRedirect(from: string, to: string) {
-          {
-            path: routes.DASHBOARD,
-            element: (
-              <ProtectedRoute scopes={[SCOPES.PERFORMANCE_READ_DASHBOARDS]}>
-                <PerformanceDashboard />
-              </ProtectedRoute>
-            )
-          },
-          {
-            path: routes.ORGANISATIONS_INDEX,
-            element: (
-              <ProtectedRoute
-                scopes={[
-                  SCOPES.ORGANISATION_READ_LOCATIONS,
-                  SCOPES.ORGANISATION_READ_LOCATIONS_MY_OFFICE,
-                  SCOPES.ORGANISATION_READ_LOCATIONS_MY_JURISDICTION
-                ]}
-              >
-                <AdministrativeLevels />
-              </ProtectedRoute>
-            )
-          },
-          { path: routes.ISSUE_COLLECTOR, element: <IssueCertificate /> },
-          { path: routes.ISSUE_VERIFY_COLLECTOR, element: <VerifyCollector /> },
-          { path: routes.ISSUE_CERTIFICATE_PAYMENT, element: <IssuePayment /> },
-          { path: routes.PRINT_RECORD, element: <PrintRecord /> },
-          {
-            path: routes.PERFORMANCE_FIELD_AGENT_LIST,
-            element: <FieldAgentList />
-          },
-          {
-            path: routes.PERFORMANCE_HOME,
-            element: (
-              <ProtectedRoute scopes={[SCOPES.PERFORMANCE_READ]}>
-                <PerformanceHome />
-              </ProtectedRoute>
-            )
-          },
-          {
-            path: routes.EVENT_COMPLETENESS_RATES,
-            element: <CompletenessRates />
-          },
-          {
-            path: routes.WORKFLOW_STATUS,
-            element: <WorkflowStatus />
-          },
-          {
-            path: routes.TEAM_SEARCH,
-            element: (
-              <ProtectedRoute
-                scopes={[
-                  SCOPES.USER_READ,
-                  SCOPES.USER_READ_MY_OFFICE,
-                  SCOPES.USER_READ_MY_JURISDICTION
-                ]}
-              >
-                <TeamSearch />
-              </ProtectedRoute>
-            )
-          },
-          {
-            path: routes.CREATE_USER_ON_LOCATION,
-            element: <CreateNewUser />
-          },
-          {
-            path: routes.CREATE_USER_SECTION,
-            element: <CreateNewUser />
-          },
-          {
-            path: routes.REVIEW_USER_FORM,
-            element: <CreateNewUser />
-          },
-          {
-            path: routes.REVIEW_USER_DETAILS,
-            element: <CreateNewUser />
-          },
-          v2RoutesConfig
-        ]
-      }
-    ]
+
+// By default we now load V2 events in the '/' route. This is a temporary flag
+// to turn off V2 events in case IET wants to switch back to V1 events.
+// This will be removed after a few releases when V2 events will be finalized.
+const turnOffV2Events =
+  typeof config.FEATURES.V2_EVENTS === 'boolean' &&
+  config.FEATURES.V2_EVENTS === false
+
+export const getRoutesConfig = (disableV2Events = turnOffV2Events) =>
+  disableV2Events
```

Comment:
Are there any changes being made in this block other than flipping the values to the ternary operator? If not then can we please keep the previous order as that would make it easier to review for any changes